Gopkg.toml

```diff
@@ -26,223 +26,39 @@ version="v0.5.0"
-
-[[override]]
-name="github.com/go-logr/logr"
-revision="9fb12b3b21c5415d16ac18dc5cd42c1cfdd40c4e"
-
-[[override]]
-name="github.com/go-logr/zapr"
-revision="7536572e8d55209135cd5e7ccf7fce43dca217ab"
-
-[[override]]
-name="github.com/gobuffalo/envy"
-version="v1.6.3"
-
-[[override]]
-name="github.com/gogo/protobuf"
-version="v1.1.1"
-
-[[override]]
-name="github.com/golang/glog"
-revision="23def4e6c14b4da8ac2ed8007337bc5eb5007998"
-
-[[override]]
-name="github.com/golang/groupcache"
-revision="66deaeb636dff1ac7d938ce666d090556056a4b0"
-
-[[override]]
-name="github.com/golang/protobuf"
-version="v1.1.0"
-
-[[override]]
-name="github.com/google/gofuzz"
-revision="24818f796faf91cd76ec7bddd72458fbced7a6c1"
-
-[[override]]
-name="github.com/googleapis/gnostic"
-version="v0.1.0"
-
-[[override]]
-name="github.com/hashicorp/golang-lru"
-revision="0fb14efe8c47ae851c0034ed7a448854d3d34cf3"
-
-[[override]]
-name="github.com/howeyc/gopass"
-revision="bf9dde6d0d2c004a008c27aaee91170c786f6db8"
-
-[[override]]
-name="github.com/hpcloud/tail"
-version="v1.0.0"
-
-[[override]]
-name="github.com/imdario/mergo"
-version="v0.3.5"
-
-[[override]]
-name="github.com/inconshreveable/mousetrap"
-version="v1.0"
-
-[[override]]
-name="github.com/joho/godotenv"
-version="v1.2.0"
-
-[[override]]
-name="github.com/json-iterator/go"
-version="1.1.4"
-
-[[override]]
-name="github.com/markbates/inflect"
-version="v1.0.0"
-
-[[override]]
-name="github.com/modern-go/concurrent"
-version="1.0.3"
-
-[[override]]
-name="github.com/modern-go/reflect2"
-version="1.0.1"
-
-[[override]]
-name="github.com/onsi/ginkgo"
-version="v1.6.0"
-
-[[override]]
-name="github.com/onsi/gomega"
-version="v1.4.1"
-
-[[override]]
-name="github.com/pkg/errors"
-version="v0.8.0"
-
-[[override]]
-name="github.com/spf13/cobra"
-version="v0.0.3"
-
-[[override]]
-name="github.com/spf13/pflag"
-version="v1.0.1"
-
-[[override]]
-name="go.uber.org/atomic"
-version="v1.3.2"
-
-[[override]]
-name="go.uber.org/multierr"
-version="v1.1.0"
-
-[[override]]
-name="go.uber.org/zap"
-version="v1.8.0"
-
-[[override]]
-name="golang.org/x/crypto"
-revision="4ec37c66abab2c7e02ae775328b2ff001c3f025a"
-
-[[override]]
-name="golang.org/x/net"
-revision="640f4622ab692b87c2f3a94265e6f579fe38263d"
-
-[[override]]
-name="golang.org/x/oauth2"
-revision="cdc340f7c179dbbfa4afd43b7614e8fcadde4269"
-
-[[override]]
-name="golang.org/x/sys"
-revision="7db1c3b1a98089d0071c84f646ff5c96aad43682"
-
-[[override]]
-name="golang.org/x/text"
-version="v0.3.0"
-
-[[override]]
-name="golang.org/x/time"
-revision="fbb02b2291d28baffd63558aa44b4b56f178d650"
-
-[[override]]
-name="golang.org/x/tools"
-revision="32950ab3be12acf6d472893021373669979907ab"
-
-[[override]]
-name="google.golang.org/appengine"
-version="v1.1.0"
-
-[[override]]
-name="gopkg.in/inf.v0"
-version="v0.9.1"
-
-[[override]]
-name="gopkg.in/tomb.v1"
-revision = "dd632973f1e7218eb1089048e0798ec9ae7dceb8"
-
-[[override]]
-name="gopkg.in/yaml.v2"
-version="v2.2.1"
+version="v4.7.1"
```

Comment:
Why the downgrade?

Reply:
That was an accident I believe, will fix